features/step_definitions/app_steps.rb

```diff
@@ -95,3 +98,49 @@
         raise "No check for the #{page_name} page."
     end
 end
+
+# HACK This is probably worthy of some award. Not the good kind.
+And /^I should see the latest post$/ do
+    sleep 2
```

Comment:
The `sleep 2` is arbitrary, but on my machine this test often fails if I don't wait for a least this amount of time.